server/controllers/courses/index.js

```diff
@@ -0,0 +1,28 @@
+const router = require('express').Router();
+
+const { courses } = require('./courses');
+const { course } = require('./course');
```

Comment:
@AbdallahAmmar96 I think these two imported function names don't match the names of the functions that were exported in courses.js and course.js.